charts/ai-service/files/download-artifacts.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
```

Comment:
We could probably add this somewhere here if it's referencible https://github.com/Unique-AG/helm-charts/tree/f55f622828a4abfe7582c88132b089e56a3b91dc/scripts

Reply:
absolutely not. the scripts of the repo have nothing to do with the scripts of a chart.
this is exactly in the right place, see https://github.com/Unique-AG/helm-charts/tree/main/charts/kong-plugins/files